### DISCUSSION TITLE
Welcome to Gammu Discussions!

### DISCUSSION THREAD

**Discussion by nijel** [gammu/gammu#545] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gatorreina** [comment]
Today I received the email below from my cellular provider informing me that my Huawei E1786 would stop working by January 2021. I currently use this device to send/receive sms messages and to receive MMS messages-- the modem receives the hex code then establishes a broadband connection and downloads the picture/file. If all 3G devices and all devices that lack the Voice over LTE technology stop working does this mean will Gammu become obsolete in the next month? I do not know of any 5G or even 4G devices --certainly not any modems-- that work with Gammu does anyone else know of any? 

Dear Richard,

We’re reaching out to warn you that the major US carriers are on the verge of shutting down their older networks in favor of newer LTE and 5G networks.

From January 2021, older devices including your E1786 lack the Voice over LTE (VoLTE) technology to connect over these newer networks and will stop working completely with most carriers. Your device could begin to experience issues even sooner.

**Reply by Armagetron** [reply]
I am not familiar with your network, however the situation in Germany is similar, 3G/UMTS is shut off in favor of 4G/5G however 2G remains operational. So, if your provider continues to support 2G you should be fine and you do not need VoLTE for SMS at all. So you should check whether 2G is available to you.

**Reply by gatorreina** [reply]
According to my provider Ting Mobile (until recently a reseller of t-mobile now a reseller of Dish) 2G will also no longer be supported, not sure if that's accurate and not sure if that's just for the US.

**Comment by hbarnard** [comment]
Hello, I'm Hugh Barnard creator of: [Cclite](https://sourceforge.net/projects/cclite/) a non-crypto mutual social credit software. As such I also use gammu-smsd to enable folks to _'pay'_ using SMS. I've discovered after some research including: [CMS error](https://stackoverflow.com/questions/21522794/atcmgs-returns-error) why my gammu-smsd and test program give  `+CMS ERROR: 302`. It's not an uninitialised message centre it's just lacking this: `AT+CSCS="GSM"` to switch coding from hex. 

So I wondered if there's a standard way (not requesting any program mods!) to put this in a gammu-smsd initialisation parameter, it needs probably to happen once only before sending?
